src/components/MessageActions/MessageActionsBox.tsx

```diff
@@ -149,6 +149,10 @@ const UnMemoizedMessageActionsBox = <
   );
 };
 
+/**
+ * MessageActionsBox - Allows the user to flag, edit or delete a message. There's also a command(`/mute`) for muting the user that wrote the message.
```

Comment:
this sounds a little funky, let's replace with "Allows a user to edit, (add the rest) a message."